### PR TITLE
myst-parser version

### PR DESCRIPTION
**Pull request by choldgraf**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
     python_requires=">=3.6",
     package_data={"myst_nb": ["_static/mystnb.css"]},
     install_requires=[
-        "myst-parser~=0.6.0",
+        "myst-parser~=0.7.0",
```
Comment (chrisjsewell) [inline setup.py]: Use 7.1 I think (there was an addition for the Jupytext functionality

**Comment by chrisjsewell** [comment]
You need to also do the pandas theme updates

**Comment by choldgraf** [comment]
good point, may as well do it here too

**Comment by choldgraf** [comment]
latest push uses the pydata theme, let's see if the docs are happy

**Pull request merged**